code/clue.js

```diff
@@ -3,21 +3,139 @@
 const mrGreen = {
   firstName: "Jacob",
   lastName: "Green",
-  color: "green",
+  color: "Green",
   description: "He has a lot of connections",
-  age: 45,
+  age: 49,
   image: "assets/green.png",
   occupation: "Entrepreneur"
-}
+};
+
+const profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "He is a tiny confused man",
+  age: 42,
+  image: "assets/plum.png",
+  occupation: "Lawyer"
+};
+
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "orange",
+  description: "Tall intelligent woman",
+  age: 33,
+  image: "assets/scarlet.png",
+  occupation: "Tech Entrepreneur"
+};
+
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "A devious person",
+  age: 53,
+  image: "assets/peacock.png",
+  occupation: "News anchor"
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "A hard headed man",
+  age: 61,
+  image: "assets/mustard.png",
+  occupation: "Retired Navy SEAL"
+};
+
+const mrsWhite = {
+  firstName: "Alice",
+  lastName: "White",
+  color: "white",
+  description: "A sweet and caring woman",
+  age: 68,
+  image: "assets/white.png",
+  occupation: "Nurse"
+};
+
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: "rope",
-  weight: 10
+  name: "Rope",
+  weight: 1.4,
+  material: "Natural fiber",
+  image: "weapon/rope.jpg" //IMG-länken till funktionen på rad 257
+}
+
+const knife = {
+  name: "Knife",
+  weight: 0.3,
+  material: "Steel and wood",
+  image: "weapon/knife.jpg"
+}
+
+const candlestick = {
+  name: "Candlestick",
+  weight: 1.2,
+  material: "Brass",
+  image: "weapon/candlestick.jpg"
+}
+
+const dumbbell = {
+  name: "Dumbbell",
+  weight: 20,
+  material: "Steel and vinyl",
+  image: "weapon/dumbbell.jpg"
+}
+
+const poison = {
+  name: "Poison",
+  weight: 0.4,
+  material: "Green fluid",
+  image: "weapon/poison.jpg"
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const axe = {
+  name: "Axe",
+  weight: 35,
+  material: "Wood and steel",
+  image: "weapon/axe.jpg"
+}
+
+const bat = {
+  name: "Bat",
+  weight: 1.5,
+  material: "Metal",
+  image: "weapon/bat.jpg"
+}
+
+const trophy = {
+  name: "Trophy",
+  weight: 2.5,
+  material: "Metal and wood",
+  image: "weapon/trophy.jpg"
+}
+
+const pistol = {
+  name: "Pistol",
+  weight: 0.8,
+  material: "Aluminium",
+  image: "weapon/pistol.jpg"
+}
+
+const weaponsImage = {
+  image: "axe.jpg",
+  image: "bat.jpg",
+  image: "candlestick",
+  image: "dumbbell.jpg",
+  image: "knife",
+  image: "pistol",
+  image: "poison"
+}
```

Comment:
Interesting. How come you having all of the images inside of the objects and then once again in its own object?

Reply:
Hi, 
I thought we where supposed to choose a specific killer so that's why I did :-)

Regarding the images. I couldn't get it to work so when it finally did, I didn't dare start erasing anything. I will go back and look at it, thanks.